config.js

```diff
@@ -41,12 +41,14 @@ module.exports = {
     , 'do'          :  undefined
 
     , 'else'        :  undefined
+    , 'export'      :  undefined
 
     , 'finally'     :  undefined
     , 'for'         :  undefined
     , 'function'    :  undefined
 
     , 'if'          :  undefined
+    , 'import'      :  undefined
```

Comment:
So we should copy out this original `config.js` into `config-es5.js` and extend it afterwards so it includes **all** possible ES6 tokens.